notebooks/integrations/llama3/rag-elastic-llama3.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
_Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 6 Line 2_
```python
from llama_index.core.node_parser import SentenceSplitter
from llama_index.core.ingestion import IngestionPipeline
```
nitpick: move the imports to where the code being used, so its obvious which cell needs which import.

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Reply:
Updated it accordingly.